packages/apollo-server-core/src/plugin/landingPage/embeddedExplorer/index.ts

```diff
@@ -0,0 +1,188 @@
+import type { ImplicitlyInstallablePlugin } from '../../../ApolloServer';
+import { printSchema } from 'graphql';
+
+export interface ApolloServerPluginLandingPageEmbeddedExplorerOptions {
+  /**
+   * By default, the embedded explorer plugin uses the latest version of the landing
+   * page published to Apollo's CDN. If you'd like to pin the current version,
+   * pass the SHA served at
+   * https://apollo-server-landing-page.cdn.apollographql.com/_latest/version.txt
+   * here.
+   */
+  version?: string;
+  /**
+   * If specified, embed the explorer of the to the Studio page
+   * for the given registered graphRef.
+   * (You need to explicitly pass this here rather than
+   * relying on the server's ApolloConfig, because if your server is publicly
+   * accessible you may not want to display the graph ref publicly.)
+   */
+  graphRef?: string;
+  /**
+   * Folks can configure their embedded Explorer with a
+   * document, variables and headers loaded in the UI.
+   */
+  document?: string;
+  variables?: Record<string, string>;
+  headers?: Record<string, string>;
+  /**
+   * We have two options for grabbing your schema for the embedded Explorer.
+   * // TODO (maya): update this comment when private graphs can be embedded
+   * If you have a Studio graph that is public, you can specify that graphRef
+   * and we will render the Explorer of your private graph using that registered schema.
+   *
+   * You also have the option to just use the schema running on this Apollo Server
+   * instance.
+   *
+   * For subgraphs & non-federated Apollo Server schemas, your server needs
```

Comment:
I'd replace this paragraph and the next few with

> If this server is a gateway, the web app will poll your server with an introspection query every 5 seconds to see if the schema has changed. You can adjust this polling interval by setting this value, or disable the polling by setting it to 0. This defaults to 0 for non-gateway servers because they have static schemas.